network/network.go

```diff
@@ -0,0 +1,9 @@
+// Provides several types
```

Comment:
rename to doc.go

Reply:
sg, also change `domain/domian.go` to `domain/doc.go`